Makefile

```
@@ -138,6 +138,17 @@ test.neon::
 		$(REGRESSION_TEST_MODE)
 	@ echo "herd7 AArch64 NEON instructions tests: OK"
 
+test:: test.sve
+test.sve::
+	@ echo
+	$(HERD_REGRESSION_TEST) \
```

maybe consider running test in parallel:
```suggestion
	$(HERD_REGRESSION_TEST) \
	-j $(J) \
```